libs/src/libs.js

```diff
@@ -315,7 +316,8 @@ class AudiusLibs {
     logger = console,
     isDebug = false,
     preferHigherPatchForPrimary = true,
-    preferHigherPatchForSecondaries = true
+    preferHigherPatchForSecondaries = true,
+    localStorage = getPlatformLocalStorage()
```

Comment:
I think this needs to be an async fn right?

Reply:
I don't think so, `getPlatformLocalStorage` is just the fallback that checks if `window` exists and uses `window.localStorage` or node localStorage

All the instances of getItem, setItem, and removeItem had to be updated to be awaited though

Reply:
u rite